iree/turbine/kernel/wave/codegen/handlers.py

```diff
@@ -622,6 +706,23 @@ def handle_minimum(lhs: Value, rhs: Value, options: WaveCompileOptions) -> OpRes
     return result
 
 
+@handle_atomic_op(atomic_min)
+def handle_atomic_min(
+    val: Value, buffer: Value, idx: list[Value], options: WaveCompileOptions
+) -> OpResult:
+    value_element_type = get_type_or_element_type(val.type)
+    atomic_kind = None
+    # Only scalars are supported currently
+    if _is_float_type(value_element_type):
+        atomic_kind = arith_d.AtomicRMWKind.minimumf
+    elif _is_integer_like_type(value_element_type) and _is_signed_or_signless_type(
+        value_element_type
+    ):
+        atomic_kind = arith_d.AtomicRMWKind.mins
```

Comment:
should add another else to fail/error out if it's neither type

Reply:
This wasn't addressed